bin/tectonics-modeling-L0/session1-aspect.ipynb

```diff
@@ -0,0 +1,2840 @@
+{
```

Comment:
<p>Just some formatting here:</p><ol><li>What matters -&gt; what matters</li><li>remove "Also:"</li><li>"Thus, the expected ... Ra", should be on a new line</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>